src/sentry/replays/usecases/ingest/__init__.py

```diff
@@ -215,6 +215,22 @@ def recording_post_processor(
     replay_event_bytes: bytes | None,
     transaction: Span,
 ) -> None:
+    if len(segment_bytes) >= 4_000_000:
+        logger.info(
+            # Logging to the sentry.replays.slow_click namespace because
+            # its the only one configured to use BigQuery at the moment.
+            #
+            # NOTE: Needs an ops request if we want to create a new dataset.
```

Comment:
Would it be worth getting this done for "general" logging?

Reply:
Yeah that's basically what we use this namespace for anyway.  Could extract this out into a global and not have to worry about comments like these anymore.